prog/dftb+/lib_reks/reksproperty.F90

```diff
@@ -670,17 +670,18 @@ subroutine getUnrelaxedTDM22_(eigenvecs, FONs, Nc, nstates, rhoXdel)
     a = Nc + 1
     b = Nc + 2
 
+    rhoXdel(:,:,:) = 0.0_dp
```

Comment:
Why is this additional zeroing needed? I think that modification from intent out to inout is enough in this routine.

Reply:
The getUnrelaxedTDM22_ routine is intent(out) :: rhoXdel(:,:,:), so either changing to inout or initializing to zero inside works. What do you prefer?

Reply:
Please change intent(out) to intent(inout).